modular_skyrat/modules/tableflip/code/flipped_table.dm

```diff
@@ -48,7 +48,7 @@
 	if(get_dir(leaving.loc, new_location) == dir)
 		return COMPONENT_ATOM_BLOCK_EXIT
 
-/obj/structure/flippedtable/CtrlShiftClick(mob/user)
+/obj/structure/flippedtable/CtrlShiftClick(mob/living/user)
 	. = ..()
 	if(!istype(user) || !user.can_interact_with(src))
```

Comment:
```suggestion
	if(!istype(user) || !user.can_interact_with(src) || isobserver(user))
```

Reply:
Interesting. I know where I can use this for a similar oversight involving ghosts where I can't use mob/living. For my own learning purposes on this particular PR though, why is this method better? I would think that living user would be preferred since it should cover edge cases like dead but not observing mobs messing with tables.